admin/spec/service/pin-generation.service.spec.js

```diff
@@ -190,6 +191,8 @@ describe('pin-generation.service', () => {
         const result = pinGenerationService.generateSchoolPassword(school)
         expect(result.pinExpiresAt).toBeDefined()
         expect(result.schoolPin.length).toBe(8)
+        expect(/[a-z]/i.test(result.schoolPin)).toBe(true)
+        expect(/[1-9]/i.test(result.schoolPin)).toBe(true)
```

Comment:
This doesn't test that you get word-number-word

Reply:
@jon-shipley updated